.github/workflows/containers.yml

```diff
@@ -0,0 +1,76 @@
+name: Upload Container image
+
+on:
+  schedule:
+  - cron: "0 */12 * * *"
+  push:
+    branches:
+     - master
+    tags:
+      - 'v*'
+  workflow_dispatch:
```

Comment:
what's this? can be dropped?

Reply:
workflow_dispatch is useful for running this workflow manually. Useful for troubleshooting or forcing the update against master.

Reply:
👍🏻